classes/emailTemplate/DAO.php

```diff
@@ -379,7 +379,18 @@ public function installAlternateEmailTemplates(int $contextId, ?string $emailKey
                 ->exists();
 
             if (!$exists) {
-                throw new Exception('Tried to install email template as an alternate to `' . $alternateTo . '`, but no default template exists with this key.');
+                trigger_error(
+                    'Tried to install email template as an alternate to `' . $alternateTo . '`, but no default template exists with this key. Installing ' . $alternateTo . ' email template first',
+                    E_USER_WARNING
+                );
+                try {
+                    $siteDao = DAORegistry::getDAO('SiteDAO'); /** @var SiteDAO $siteDao */
+                    $site = $siteDao->getSite(); /** @var Site $site */
+                    $locales = $site->getInstalledLocales();
```

Comment:
Looks like these lines can be dropped, if you pass an empty array at the `$locales` argument, it will have the same effect.